### PR TITLE
feat: multi pivot chart

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #1537 

### Description:

Screenshot 2022-10-24 at 17 40 34

### CODE REVIEW

File: packages/frontend/src/components/ChartConfigPanel/FieldLayoutOptions.tsx [hunk 1]
```diff
@@ -173,33 +189,80 @@ const FieldLayoutOptions: FC<Props> = ({ items }) => {
...
+                                        placeholder="Select a field to group by"
+                                        activeField={
+                                            chartHasMetricOrTableCalc
+                                                ? groupSelectedField
+                                                : undefined
+                                        }
+                                        onChange={(item) => {
+                                            setPivotDimensions(
+                                                pivotDimensions
+                                                    ? pivotDimensions.map(
```
Comment (rephus) [inline packages/frontend/src/components/ChartConfigPanel/FieldLayoutOptions.tsx:219]: I don't understand what this is supposed to do...

**Comment by rephus** [comment]
I tested it and it think it looks great :100:

**Review by rephus — APPROVED** [review]
LGTM

File: packages/frontend/src/components/ChartConfigPanel/FieldLayoutOptions.tsx (hunk 1 above)
Comment (ZeRego) [reply packages/frontend/src/components/ChartConfigPanel/FieldLayoutOptions.tsx]: It is replacing the value in the array. I moved that logic to a util function so is readable 👍

**Comment by TuringLovesDeathMetal** [comment]
ahhhh! @ZeRego this is so good 😍 I feel like the next steps will be being able to drag-and-drop to reorder the groups, but I'll open an issue for that :)

**Pull request merged**